assets/json/fish.json

```diff
@@ -0,0 +1,580 @@
+[
+  {
+    "name": "Charc Eel",
+    "thumb": "img/fish/charceel.png",
+    "unique": {
+      "name": "Charc Electroplax",
+      "thumb": "img/fish/part/electroplax.png",
+      "wiki": "http://warframe.wikia.com/wiki/Charc_Electroplax"
+    },
+    "resources": {
+      "small": [
+        3,
```

Comment:
It's not intuitive what this 0-indexed array maps out to, and the various sizes are repeated in the `standings` object, which indicates an issue with the model.  Instead of the `resources` and `standings` being children of the fish's root, I propose this under the root:

```javascript
{
  "sizes": {
    "small": {
      "meat": 3,
      "scales": 2,
      "oil": 1,
      "standing": 25
    },
    "medium": {
      "...": "..."
    },
    "large": {
      "...": "..."
    }
  }
}
```

If it's important to distinguish between resources and standing, then have those as a property of each object in `sizes`:

```javascript
{
  "small": {
    "resources": {
      "meat": 3,
      "scales": 2,
      "oil": 1
    },
    "standing": 25
  }
}
```

Reply:
and this will make even less on the boot object

Reply:
if i'm going do the sizes on the root, i'm not going to put resources another level down, it's going to get unnecessarily deep.

Reply:
The accuracy of the model is more important than how deep you go.  We essentially embed a 3x4 table on each row, so we have to go that deep.  The boot's only "size" can be `"-"` (or if you want to match what it currently is on production, `"Boot"`).

Reply:
this doesn't make it any less accurate. and no, that's actually not what we're doing. they're still individual cells on the same table. there's no embedded tables.

Reply:
It does make the model less accurate because it inverts the ownership of your objects.  The size of a fish determines your yield; sizes are not a property of a resource or standing.

I said _essentially_ because while _technically_ we're not using separate HTML `<table>` elements, UI-wise we _are_ embedding a table inside the columns after the fish name, as can be seen below:

![screenshot 2018-07-26 15 31 08](https://user-images.githubusercontent.com/5075438/43284435-c74fe7e2-90e9-11e8-8d15-a987a4afe663.png)

Reply:
@dsinn refactored.